lib/rules/no-mixed-spaces-and-tabs.js

```diff
@@ -26,13 +29,57 @@ module.exports = function(context) {
     var COMMENT_START = /^\s*\/\*/,
         MAYBE_COMMENT = /^\s*\*/;
 
+    /**
+     * Determines if a given line and column are contained within a location.
+     * @param {Location} loc The location object from an AST node.
+     * @param {int} line The line to check.
+     * @param {int} column The column to check.
+     * @returns {boolean} True if the line and column are inside the location, false if not.
+     * @private
+     */
+    function inLoc(loc, line, column) {
```

Comment:
Thought about that, but didn't see an obvious other use for it. Will keep an eye out going forward.